docker/data-science/notebooks/ch06/3.spell-correction.ipynb

```diff
@@ -5320,133 +248,26 @@
-       "      <td>carolina</td>\n",
-       "      <td>5</td>\n",
-       "      <td>8</td>\n",
-       "      <td>c</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>1</th>\n",
-       "      <td>laura</td>\n",
-       "      <td>5</td>\n",
-       "      <td>5</td>\n",
-       "      <td>l</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>2</th>\n",
-       "      <td>marling</td>\n",
-       "      <td>5</td>\n",
-       "      <td>7</td>\n",
-       "      <td>m</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>3</th>\n",
-       "      <td>hubs</td>\n",
-       "      <td>6</td>\n",
-       "      <td>4</td>\n",
-       "      <td>h</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>4</th>\n",
-       "      <td>gorillaz</td>\n",
-       "      <td>6</td>\n",
-       "      <td>8</td>\n",
-       "      <td>g</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>5</th>\n",
-       "      <td>tosheba</td>\n",
-       "      <td>6</td>\n",
-       "      <td>7</td>\n",
-       "      <td>t</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>6</th>\n",
-       "      <td>tenacious</td>\n",
-       "      <td>5</td>\n",
-       "      <td>9</td>\n",
-       "      <td>t</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>7</th>\n",
-       "      <td>mute</td>\n",
-       "      <td>6</td>\n",
-       "      <td>4</td>\n",
-       "      <td>m</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>8</th>\n",
-       "      <td>math</td>\n",
-       "      <td>6</td>\n",
-       "      <td>4</td>\n",
-       "      <td>m</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>9</th>\n",
-       "      <td>eragon</td>\n",
-       "      <td>6</td>\n",
-       "      <td>6</td>\n",
-       "      <td>e</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "    misspell  misspell_counts  misspell_length initial\n",
-       "0   carolina                5                8       c\n",
-       "1      laura                5                5       l\n",
-       "2    marling                5                7       m\n",
-       "3       hubs                6                4       h\n",
-       "4   gorillaz                6                8       g\n",
-       "5    tosheba                6                7       t\n",
-       "6  tenacious                5                9       t\n",
-       "7       mute                6                4       m\n",
-       "8       math                6                4       m\n",
-       "9     eragon                6                6       e"
-      ]
-     },
-     "execution_count": 16,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "AttributeError",
+     "evalue": "'DataFrame' object has no attribute 'orderBy'",
```

Comment:
Note: error in SQL. Need to run this cell and verify output or otherwise fix code if still breaking.